### PR TITLE
Feature/refactor

### PR DESCRIPTION
**Pull request by magec**
# WAT
Renamed class name and table name to make it more convention aware.

### CODE REVIEW

File: spec/dummy/log/development.log [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-  [1m[36m (0.2ms)[0m  [1mCREATE DATABASE `nps_surveys_test` DEFAULT CHARACTER SET `utf8` COLLATE `utf8_unicode_ci`[0m
-  [1m[35m (29.2ms)[0m  CREATE TABLE `surveys_responses` (`id` int(11) DEFAULT NULL auto_increment PRIMARY KEY, `survey` varchar(255) NOT NULL, `user_id` int(11) NOT NULL, `score` int(11), `feedback` text, `created_at` datetime NOT NULL, `updated_at` datetime NOT NULL) ENGINE=InnoDB
-  [1m[36m (27.7ms)[0m  [1mCREATE TABLE `users` (`id` int(11) DEFAULT NULL auto_increment PRIMARY KEY, `login` varchar(255), `password` varchar(255), `created_at` datetime NOT NULL, `updated_at` datetime NOT NULL) ENGINE=InnoDB[0m
-  [1m[35m (25.8ms)[0m  CREATE TABLE `schema_migrations` (`version` varchar(255) NOT NULL) ENGINE=InnoDB
-  [1m[36m (52.3ms)[0m  [1mCREATE UNIQUE INDEX `unique_schema_migrations` ON `schema_migrations` (`version`)[0m
-  [1m[35m (0.2ms)[0m  SELECT version FROM `schema_migrations`
-  [1m[36m (5.0ms)[0m  [1mINSERT INTO `schema_migrations` (version) VALUES ('20160721092137')[0m
-  [1m[35m (4.6ms)[0m  INSERT INTO `schema_migrations` (version) VALUES ('20160614145840')
```
Comment (sauloperez) [inline spec/dummy/log/development.log]: we should completely gitignore this log/ folder

**Comment by sauloperez** [comment]
Looks good

**Comment by masylum** [comment]
👍

File: spec/dummy/log/development.log (hunk 1 above)
Comment (magec) [inline spec/dummy/log/development.log]: It is already ignored, but the file was there before the ignore.

**Comment by enricostano** [comment]
:+1:

**Pull request merged**